pages/home.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import AppNav from '../components/AppNav'
+import LandingPage from '../components/LandingPage'
+import Footer from '../components/Footer'
+
+const Home: React.FC = () => (
+  <>
+    <AppNav />
```

Comment:
move landing page nav out, and put here directly. Landing page Nav shouldn't show up anywhere else imo.

Reply:
nvm, saw the logic.